datafusion/core/src/test_util/mod.rs

```diff
@@ -55,6 +55,7 @@ use tempfile::TempDir;
 #[cfg(feature = "parquet")]
 pub use datafusion_common::test_util::parquet_test_data;
 pub use datafusion_common::test_util::{arrow_test_data, get_data_dir};
+use datafusion_physical_expr::EquivalenceProperties;
```

Comment:
Let's move this import statement up so it doesn't get mixed up with the backwards compatibility imports.